js/ToDo.js

```diff
@@ -0,0 +1,305 @@
+window.addEventListener('load',function(){
+    
+
+// variables & return value
+
+// 할 일 입력란
+    // typeBox_Todo
+        var typeBox_TD = document.querySelector('#typeBox_Todo');
+
+    // typeBox_Todo value
+        function typeBox_TD_txt() { // 유저가 input에 입력한 ToDo
+            return document.querySelector('#typeBox_Todo').value;
+        }
```

Comment:
document.querySelector('#typeBox_Todo')를 변수로 사용하고 있으므로 아래와 같이 수정한다면 보다 짧게 코드를 작성하며 훨씬 더 쉽게 이해할수 있도록 만들수 있음
```suggestion
        var typeBox_TD = document.querySelector('#typeBox_Todo');

    // typeBox_Todo value
        function typeBox_TD_txt() { // 유저가 input에 입력한 ToDo
            return typeBox_TD.value;
        }
```